R/utils.R

```diff
@@ -427,7 +355,7 @@ generateSimulations <- function(model,
                 params_list,
                 lapply(param_iter, "[[", i)
             )
-            simulations_local <- generateSimulations(
+            simulations_local <- .generateSimulations(
```

Comment:
are you sure that generateSimulations is not an exported function needed elsewhere in case studies?

Reply:
Yes, it is not a part of any simulation function. It is an auxiliary function for miaSim